README.md

```diff
@@ -619,19 +620,25 @@ Usually, this means excluding a celebration defined in `src/calendars/general.js
 }
 ```
 
-An example of this can be seen in the national calendar of Slovakia (`src/calendars/slovakia.js`) where the celebrations of Shrove Monday and Shrove Tuesday are not relevant in the calendar output. 
+An example of this can be seen in the national calendar of Slovakia (`src/calendars/slovakia.js`) where the celebrations of Shrove Monday and Shrove Tuesday are not relevant in the calendar output.
 
-Therefore, the `src/calendars/slovakia.js` redefines these 2 celebration keys with only one property, `drop` with the value of `true` so that they are excluded in the `calendarFor` output. 
+Therefore, the `src/calendars/slovakia.js` redefines these 2 celebration keys with only one property, `drop` with the value of `true` so that they are excluded in the `calendarFor` output.
 
 Note: When defining `drop`, only the key of the celebration is mandatory. Other keys do not have to be defined. `drop` operations also have higher precedence than overriding (meaning, they are run first before prioritization logic).
 
 ## Localizing celebration names <a name="localisation"></a>
 
-Celebration names in Romcal can be localized to any language that is already supported by [Moment i18n](http://momentjs.com/docs/#/i18n/). 
+Celebration names in Romcal can be localized to any language that is already supported by [Moment i18n](http://momentjs.com/docs/#/i18n/).
+
+Locales are stored as `.js` files in the `src/locales` directory.
+
+If the given locale contain a region (the second group of letters after a dash, so for `fr-CA`: `CA` represent the region), a base language will be set as a locale fallback (`fr` in our example).
```

Comment:
1. _contain_ → _contains_

2. Consider: _dash_ → _hyphen_; I know that everybody, even native English-speaking people) use _dash_ instead of hyphen, but _hyphen_ is short (-), n-dash is wider (–; wide approx as _n_) and m-dash is even wider (—; wide approx as _m_).

> If the given locale contains a region (the second group of letters after a hyphen, e.g. for `fr-CA`, the `CA` represents the region), a base language will be set as a locale fallback (`fr` in our example).